src/toady/cli.py

```
@@ -251,20 +251,33 @@ def fetch(
     is_flag=True,
     help="Output in human-readable format instead of JSON",
 )
+@click.option(
+    "--verbose",
+    "-v",
+    is_flag=True,
+    help="Show additional details about the reply and context",
+)
 @click.pass_context
-def reply(ctx: click.Context, comment_id: str, body: str, pretty: bool) -> None:
+def reply(
```

Pylint (3.3.7)

[refactor] Too many branches (53/12)

(R0912)